mlflow/ml-package-versions.yml

```
@@ -42,6 +42,17 @@ pytorch:
     run: |
       pytest tests/pytorch/test_pytorch_model_export.py tests/pytorch/test_pytorch_metric_value_conversion_utils.py
 
+torch:
+  package_info:
+    pip_release: "torch"
+    install_dev: |
+      export PACKAGE_NAME=pytorch
+      pip install git+https://github.com/PytorchLightning/pytorch-lightning.git
+  autologging:
+    minimum: "1.4"
+    maximum: "1.14"
```

```suggestion
    minimum: "1.4.0"
    maximum: "1.13.1"
```

This should fix the error.

---

`set_matrix.py` checks whether the specified versions exist on PyPI. `1.4` or `1.14` don't, hence the error.